back/src/data/Repository/RepoCollab.ts

```diff
@@ -24,7 +24,7 @@ export class RepoCollab implements IRepoCollab {
   }
 
   async findAll(): Promise<Collab[]> {
```

Comment:
Rename this class to "CollabsRepo", putting the business first and the technical type after please